treeherder/webapp/api/job_log_url.py

```diff
@@ -45,37 +45,3 @@ def list(self, request, project, jm):
         job_log_url_list = jm.get_job_log_url_list([job_id])
 
         return Response(job_log_url_list)
-
-    @detail_route(methods=['post'])
-    @with_jobs
-    def update_parse_status(self, request, project, jm, pk=None):
-        """
-        Change the log parsing status for a log reference (eg from 'pending' to 'parsed').
-        """
-        try:
-            parse_status = request.data["parse_status"]
-        except KeyError:
-            raise ParseError(detail=("The parse_status parameter is mandatory for this endpoint"))
-        jm.update_job_log_url_status(pk, parse_status)
-        obj = jm.get_job_log_url_detail(pk)
-        return Response(obj)
-
-    @detail_route(methods=['post'])
-    @with_jobs
-    def parse(self, request, project, jm, pk=None):
```

Comment:
The `parse` endpoint is used by the UI:
https://github.com/mozilla/treeherder/blob/5c5f24e3b3c794897629a203113a9e67f7f40657/ui/js/models/job_log_url.js#L18

Whether we still need that (bug 1076769) is another matter :-)